src/engraving/rendering/dev/chordlayout.cpp

```diff
@@ -1514,7 +1515,18 @@ void ChordLayout::layoutChords1(LayoutContext& ctx, Segment* segment, staff_idx_
         skipAccidentals(segment, startTrack, endTrack);
     }
 
-    if (staff && staff->isTabStaff(tick) && (!staff->staffType() || !staff->staffType()->stemThrough())) {
+    const bool isSimpleTab = !staff->staffType() || (!staff->staffType()->stemThrough() && !staff->staffType()->isCommonTabStaff());
+    if (staff && staff->isTabStaff(tick) && isSimpleTab) {
+        // Reset position
+        for (track_idx_t track = partStartTrack; track < partEndTrack; ++track) {
```

Comment:
Could you explain why you need to do this reset here? Do chords in normal staves also perform this reset? If so, it would be better to perform it in the same place if we can

Reply:
This is needed when switching from full/common TAB to simple TAB.  It isn't needed for normal staves as these always have offsets calculated and applied later on in `layoutChords1`.  I'll leave a comment.